src/v3/src/transformer/phone/phoneChallengeTransformer.ts

```diff
@@ -89,7 +83,9 @@ export const transformPhoneChallenge: IdxStepTransformer = ({ transaction, formB
 
   uischema.elements.push(submitButtonControl);
   uischema.elements.unshift(carrierChargeDisclaimerText);
-  uischema.elements.unshift(informationalText);
+  if (typeof subtitleElement !== 'undefined') {
+    uischema.elements.unshift(subtitleElement);
+  }
```

Comment:
consider inlining the logic above down here so you don't have to check for `undefined`